sycl/include/sycl/handler.hpp

```diff
@@ -438,14 +438,22 @@ class __SYCL_EXPORT handler {
   ///        is null if no secondary queue is associated with the submission.
   /// \param CallerNeedsEvent indicates if the event resulting from this handler
   ///        is needed by the caller.
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
+  // TODO: This function is not used anymore, remove it in the next
+  // ABI-breaking window.
   handler(std::shared_ptr<detail::queue_impl> Queue,
           std::shared_ptr<detail::queue_impl> PrimaryQueue,
           std::shared_ptr<detail::queue_impl> SecondaryQueue,
           bool CallerNeedsEvent);
+#endif
+  __SYCL_DLL_LOCAL handler(std::shared_ptr<detail::queue_impl> Queue,
```

Comment:
`__SYCL_DLL_LOCAL` is empty for Windows. Is it a way to make a symbol local for Windows as well?